crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -113,6 +113,10 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
+    pub fn from_entities(queue: &'s mut CommandQueue, entities: &'w Entities) -> Self {
```

Comment:
Just noticed that `Commands` already has the function `new_from_entities(&'s mut CommandQueue, &'w Entities)`. Can you just use that instead?